scripts/variantstore/wdl/extract/create_variant_annotation_table.py

```diff
@@ -143,6 +182,7 @@ def make_annotated_json_row(row_position, variant_line, transcript_line):
         nirvana_gnomad_fieldname = vat_nirvana_gnomad_dictionary.get(vat_gnomad_fieldname)
         gnomad_fieldvalue = variant_line.get("gnomad").get(nirvana_gnomad_fieldname)
         row[vat_gnomad_fieldname] = gnomad_fieldvalue
+      row = get_gnomad_subpop(variant_line["gnomad"], row) # TODO I dont like this---lets create a new one and merge it, not pass the row thru
```

Comment:
I agree — if possible, have `get_gnomad_subpop` just return the new information and add it to `row`.